src/features/HomeContents/components/Base/index.tsx

```diff
@@ -1,17 +1,29 @@
+import { format } from 'date-fns'
+
 import * as s from './style.css'
 
 import FakeTimetable from '@/features/HomeContents/components/FakeTimetable'
 import RecommendedLecture from '@/features/HomeContents/components/RecommendedLecture'
+import HomeContentsSchedule from '@/features/HomeContents/components/Schedule'
 import { Typography } from '@/ui/Typography'
+import { useAuth } from '@/util/auth/useAuth'
 
 const HomeContentsBase = () => {
+  const { authState } = useAuth()
+  const showSchedule = authState
```

Comment:
p5;
취향 차이일 거 같긴 한데, 이 경우에는 isLogin으로 이름지어주는게
ui 논리 플로우를 읽는 데 더 명확할 거 같다는 갠적인 생각이 있습니당

```suggestion
  const isLogin = authState
```